public/404.html

```diff
@@ -1,49 +0,0 @@
-<!--  License: CC0-1.0 -->
```

Comment:
I'd like the new code error pages styled somewhat like these original pages. Here's what it needs:

* the title and appropriate error code
* the body looking relatively similar but:
  *  use the appropriate erb and rails view functionality using [View Helpers](https://guides.rubyonrails.org/v6.1/action_view_helpers.html) and other necessary parts of ActionView.
  * use  `image_tag` for [including the image](https://guides.rubyonrails.org/v6.1/action_view_helpers.html#assettaghelper). This means the SVG will have to be in the appropriate place in the `app/assets` directory (Also, make sure to add an `alt` description to that image; it should have been there in the first place.
  
  *move to the head of the document like it's supposed to be.
  
 * create and use a layout just for ErrorController. The goal is to be able to reuse everything between all of the error pages except the error messages unique to each error page.